integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/ReceiptPolling.java

```diff
@@ -49,14 +57,29 @@ public void checkReceiptStatus() {
 
         receipts.forEach(r -> {
             log.info(markerFrom(r), "Checking status, messageId={}", r.getMessageId());
-            ReceiptStrategy strategy = ReceiptStrategyFactory.getFactory(r);
+            ReceiptStrategy strategy = receiptStrategyFactory.getFactory(r);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this instance-reference to a static reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS2209)